nodestream_plugin_neptune/ingest_query_builder.py

```diff
@@ -111,7 +107,7 @@ def _to_string_values(props: dict):
     for k, v in props.items():
         # Neptune can handles datetime type
         # see: https://docs.aws.amazon.com/neptune/latest/userguide/feature-opencypher-compliance.html#opencypher-compliance-differences
-        if isinstance(v, Timestamp):
+        if isinstance(v, Timestamp) or isinstance(v, datetime):
```

Comment:
```suggestion
        if isinstance(v, (Timestamp, datetime)):
```

This is a slightly more brief version of the same check.